test/integration/concrete/lockup/withdraw-max-and-transfer/withdrawMaxAndTransfer.tree

```diff
@@ -22,4 +22,5 @@ withdrawMaxAndTransfer.t.sol
                   ├── it should transfer the NFT
                   ├── it should emit a {WithdrawFromLockupStream} event
                   ├── it should emit a {Transfer} event
-                  └── it should emit a {MetadataUpdate} event
+                  ├── it should emit a {MetadataUpdate} event
+                  └── it should return the withdrawn amount
```

Comment:
In the source code, it returns `withdrawableAmountOf` so it should `it should return the withdrawable amount`.